lib/vast/Conversion/Common/IRsToLLVM.cpp

```diff
@@ -864,32 +864,80 @@ namespace vast::conv::irstollvm
         logical_not
     >;
 
+    struct minus : base_pattern< hl::MinusOp >
+    {
+        using base = base_pattern< hl::MinusOp >;
+        using base::base;
+
+        logical_result matchAndRewrite(
+                    hl::MinusOp op, hl::MinusOp::Adaptor adaptor,
+                    conversion_rewriter &rewriter) const override
+        {
+            auto arg = adaptor.getArg();
+            if (is_lvalue(arg))
+                return logical_result::failure();
+            auto arg_type = type_converter().convertType(arg.getType());
+
+            auto zero = this->constant(rewriter, op.getLoc(), arg_type, 0);
+
+            if (llvm::isa< mlir::FloatType >(arg_type))
+                rewriter.replaceOpWithNewOp< LLVM::FSubOp >(op, zero, arg);
+            else
+                rewriter.replaceOpWithNewOp< LLVM::SubOp >(op, zero, arg);
+
+            return logical_result::success();
+        }
+    };
+
+    struct plus : base_pattern< hl::PlusOp >
```

Comment:
Is this not simply a case of `one_to_one` conversion we have a base helper for?

Reply:
Nope, the plus op does literally nothing. It's for example:
```
int a = 5;
int b = +a;
```
It is present in the AST, but it isn't present in the LLVM IR afterwards.

Reply:
`ignore_pattern` base helper then?

Reply:
Oh… Yeah, that does it.